### PR TITLE
Macast: Add version 0.7

### PR DESCRIPTION
**Pull request by happytommyl**
Closes to #7758

- [-] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/macast.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "64bit": {
+            "url": "https://github.com/xfangfang/Macast/releases/download/v0.7/Macast-Windows-v0.7.exe"
+            "hash": "sha1:6400b23f29232af36f20ae1586ad28b05e5a79e9"
+        }
+    },
+    "bin": [["Macast-Windows-v0.7.exe", "macast" ]],
+    "shortcuts": [
+        [
+            "Macast-Windows-v0.7.exe",
+            "macast"
```
Comment (Lutra-Fs) [inline bucket/macast.json:16]: "Macast" is better for an entry in start menu.

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]

**Review by Lutra-Fs — APPROVED** [review]

File: bucket/macast.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.7",
+    "description": "A menu bar application using mpv as DLNA Media Renderer. You can push videos, pictures or musics from your mobile phone to your computer.",
```
Comment (rashil2000) [inline bucket/macast.json:3]:
```suggestion
    "description": "A tool which uses mpv as DLNA Media Renderer and allows you to push videos, pictures or music from your mobile phone to your computer.",
```

File: bucket/macast.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "homepage": "https://github.com/xfangfang/Macast",
+    "license": "GNU General Public License v3.0",
```
Comment (rashil2000) [inline bucket/macast.json:5]:
```suggestion
    "license": "GPL-3.0-only",
```

File: bucket/macast.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/xfangfang/Macast/releases/download/v0.7/Macast-Windows-v0.7.exe",
+            "hash": "sha1:6400b23f29232af36f20ae1586ad28b05e5a79e9"
```
Comment (rashil2000) [inline bucket/macast.json:9]: Use sha256

File: bucket/macast.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    },
+    "bin": [["Macast-Windows-v0.7.exe", "macast" ]],
```
Comment (rashil2000) [inline bucket/macast.json:12]: Commandline arguments?

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Is Macast-Windows-v0.7.exe an installer or a direct executable?

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]

**Comment by Lutra-Fs** [comment]
> Is Macast-Windows-v0.7.exe an installer or a direct executable?

it is a direct executable file

File: bucket/macast.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.7",
+    "description": "A menu bar application using mpv as DLNA Media Renderer. You can push videos, pictures or musics from your mobile phone to your computer.",
+    "homepage": "https://github.com/xfangfang/Macast",
+    "license": "GNU General Public License v3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/xfangfang/Macast/releases/download/v0.7/Macast-Windows-v0.7.exe",
```
Comment (rashil2000) [inline bucket/macast.json:8]:
```suggestion
            "url": "https://github.com/xfangfang/Macast/releases/download/v0.7/Macast-Windows-v0.7.exe#/macast.exe",
```

File: bucket/macast.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+            "hash": "sha1:6400b23f29232af36f20ae1586ad28b05e5a79e9"
+        }
+    },
+    "bin": [["Macast-Windows-v0.7.exe", "macast" ]],
+    "shortcuts": [
+        [
+            "Macast-Windows-v0.7.exe",
```
Comment (rashil2000) [inline bucket/macast.json:15]:
```suggestion
            "macast.exe",
```

File: bucket/macast.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+            "Macast"
+        ]
+    ],
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/xfangfang/Macast/releases/download/v$version/Macast-Windows-v$version.exe"
```
Comment (rashil2000) [inline bucket/macast.json:22]:
```suggestion
                "url": "https://github.com/xfangfang/Macast/releases/download/v$version/Macast-Windows-v$version.exe#/macast.exe"
```

File: bucket/macast.json (hunk 5 above)
Comment (rashil2000) [reply bucket/macast.json:12]: @Lutra-Fs what about this?

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/macast.json (hunk 5 above)
Comment (Lutra-Fs) [reply bucket/macast.json:12]: if we do like the following changes then line 12 should be changed as 
``` "bin":"macast.exe", ```

> > Commandline arguments?
> 
> @Lutra-Fs what about this?

Sry, I am a little bit confused what you mean.

Comment (Lutra-Fs) [reply bucket/macast.json:12]: the software is not CLI software.

Comment (rashil2000) [reply bucket/macast.json:12]: > the software is not CLI software.

Then we can just remove this line

File: bucket/macast.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    "description": "A tool which uses mpv as DLNA Media Renderer and allows you to push videos, pictures or music from your mobile phone to your computer.",
+    "homepage": "https://github.com/xfangfang/Macast",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/xfangfang/Macast/releases/download/v0.7/Macast-Windows-v0.7.exe#/macast.exe",
+            "hash": "sha1:6400b23f29232af36f20ae1586ad28b05e5a79e9"
+        }
+    },
+    "bin": [["macast" ]],
```
Comment (rashil2000) [inline bucket/macast.json:12]:
```suggestion
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**